src/libraries/Microsoft.PowerFx.Interpreter/UDF/UDFLazyBinder.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    internal class UDFLazyBinder
+    {
+        private readonly UserDefinedTexlFunction _function;
+        
+        private readonly List<ExpressionError> _expressionError = new List<ExpressionError>();
+
+        public string Name
```

Comment:
nit, can just be `public readonly string Name`

Reply:
Changed.